onnx/defs/tensor/defs.cc

```diff
@@ -685,9 +685,14 @@ ONNX_OPERATOR_SET_SCHEMA(
             return;
           }
 
-          int r = ctx.getInputType(0)->tensor_type().shape().dim_size();
-          int q = ctx.getInputType(1)->tensor_type().shape().dim_size();
-
+          auto& data_shape = ctx.getInputType(0)->tensor_type().shape();
+          auto& indices_shape = ctx.getInputType(1)->tensor_type().shape();
```

Comment:
Is it possible it doesn't have a shape?

Reply:
I believe this is already checked by `hasNInputShapes(ctx, 2)`